src/components/timeline/TrackPowerGraph.js

```diff
@@ -49,8 +49,8 @@ type CanvasProps = {|
   +rangeStart: Milliseconds,
   +rangeEnd: Milliseconds,
   +counter: Counter,
-  +counterSampleRanges: Array<[IndexIntoSamplesTable, IndexIntoSamplesTable]>,
-  +maxCounterSampleCountsPerMs: number[],
+  +counterSampleRange: [IndexIntoSamplesTable, IndexIntoSamplesTable],
+  +maxCounterSampleCountsPerMs: number,
```

Comment:
This prop is still using sample "counts" with the plural count. I think it should be named as `maxCounterSampleCountPerMs`. And on the rest of the patch. What do you think?